project/styles/components/section.scss

```
@@ -1282,3 +1307,25 @@
 .section--card-business {
   @include section-inner(12);
 }
+
+/**
+ * button-bar
+ */
+
+.section--buttonbar {
+  @include section-inner(12);
+
+  .section__actions {
```

ik denk dat je voor de situatie waar er alleen actions zijn, geen content of eyebrow/title, je de margin aan de bovenkant er nog af moeten halen. Dan blijft de section-spacing kloppen. Even uit m'n hoofd, zoiets:

```suggestion
  .section__actions {
    &:only-child {
      @include section-margin-top(none);
    }
```

---

aanpassing is gedaan (handmatig). Ik zie in de code dat margin-top op 0 is gezet, ik zie visueel geen verschil.